### PR TITLE
JavaScript section to Markdown - French

### PR DESCRIPTION
**Pull request by SphinxKnight**
Let's do this.

@tristantheb could I ask a favor of you to review how it plays on your side?

@wbamberg: this looks quite analogous to https://github.com/mdn/content/pull/7092, I hope I didn't miss a thing. Let me know if you spot something.

Last but least, @fiji-flo could you confirm or deny that https://github.com/mdn/yari/pull/4251 needs to be merged before this one?

### CODE REVIEW

**Comment by SphinxKnight** [comment]
[Conversion report](https://github.com/mdn/translated-content/files/6913094/md-conversion-problems-report-2021-08-01T17.16.31.939Z.md)

**Comment by wbamberg** [comment]
Thanks for this @SphinxKnight . Great work!

We should fix h2m so it doesn't just list the top 20 but lists all unconvertibles. As it is, there's likely to be a long tail of single-occurrence problems that don't show in the summary. I was going to try to patch h2m, then run this conversion again to get a new report, but it seems like [it's complicated](https://github.com/mdn/translated-content/discussions/1236#discussioncomment-1116059). Sorry it's complicated - it's hard to be the first person to try out a process :(.

But anyway, some little things I saw in the report:

* there are several `p.hidden` elements that are not being  converted, such as: https://github.com/mdn/translated-content/blob/86bfc6444602934695b25cee06320c49946e513c/files/fr/web/javascript/reference/global_objects/map/%40%40iterator/index.html#L21


* there are three `` elements that are not being converted because they contain inline styles, such as: https://github.com/mdn/translated-content/blob/86bfc6444602934695b25cee06320c49946e513c/files/fr/web/javascript/eventloop/index.html#L21
* there's a `div.callout` that's not being converted, probably because our converter wants callouts to start with an H4. I filed a PR which ought to address that (https://github.com/mdn/yari/pull/4363), but should this be a callout or a note? What even is the different  between a note and a callout anyway? https://github.com/mdn/translated-content/blob/86bfc6444602934695b25cee06320c49946e513c/files/fr/web/javascript/reference/functions/arguments/index.html#L29-L31
* there's a `<code>` element that's not being converted because it contains internal markup: https://github.com/mdn/translated-content/blob/86bfc6444602934695b25cee06320c49946e513c/files/fr/web/javascript/reference/global_objects/typedarray/from/index.html#L17


* there's a broken `<dl>` that's not being converted: https://github.com/mdn/translated-content/blob/86bfc6444602934695b25cee06320c49946e513c/files/fr/web/javascript/reference/functions/index.html#L330-L332.

**Comment by SphinxKnight** [comment]
Thanks a lot @wbamberg :) I should have paid more attention to the report :x I'll fix those in the coming hours and reapply the changes here.

**Comment by SphinxKnight** [comment]
Take n°2

**Comment by SphinxKnight** [comment]
[Conversion report](https://github.com/mdn/translated-content/files/6921399/md-conversion-problems-report-2021-08-03T06.02.30.328Z.md)

**Review by tristantheb — APPROVED** [review]

File: files/fr/web/javascript/enumerability_and_ownership_of_properties/index.md [hunk 1]
```diff
@@ -0,0 +1,495 @@
...
+                <code
+                  ><a
+                    href="/fr/docs/Web/JavaScript/Reference/Opérateurs/L_opérateur_in"
+                    >in</a
+                  ></code
+                >
+              </td>
+            </tr>
+          </tbody>
+        </table>
```
Comment (tristantheb) [inline files/fr/web/javascript/enumerability_and_ownership_of_properties/index.md:116]: Tableau non converti (+ quelques lignes en dessous de la sélection)

File: files/fr/web/javascript/enumerability_and_ownership_of_properties/index.md [hunk 2]
```diff
@@ -0,0 +1,495 @@
...
+      <td>false</td>
+      <td>false</td>
+      <td>false</td>
+      <td>false</td>
+      <td>false</td>
+      <td>false</td>
+      <td>false</td>
+    </tr>
+  </tbody>
+</table>
```
Comment (tristantheb) [inline files/fr/web/javascript/enumerability_and_ownership_of_properties/index.md:485]: Tableau non converti

**Review by tristantheb — CHANGES_REQUESTED** [review]
Il y a beaucoup de tableaux non convertis, on a un format de définit pour ou pas encore ? (Et git lag à mot avec la PR, ça va être LOOONG à tout vérifier xD)

File: files/fr/web/javascript/enumerability_and_ownership_of_properties/index.md (hunk 2 above)
Comment (SphinxKnight) [reply files/fr/web/javascript/enumerability_and_ownership_of_properties/index.md:485]: @tristantheb https://raw.githubusercontent.com/mdn/content/main/files/en-us/web/javascript/enumerability_and_ownership_of_properties/index.md "works for me" I guess ^ - ^

**Review by tristantheb — APPROVED** [review]
Bon ah let's go :D

**Comment by SphinxKnight** [comment]
Since https://github.com/mdn/yari/pull/4251 has been merged, let's merge this one as well :)

**Pull request merged**